_data/comments/test-markdown/comment-1545281843190.yml

```
@@ -0,0 +1,7 @@
+_id: bc9dd1e0-0413-11e9-beb4-d1f35c904fc1
+message: "Test again\r\n\r\n> Markdown work?"
+name: Vincent Tam
+email: 4980f828b9588a712cc2013ac10fd13a
+url: 'https://lstu.fr/3s'
```

Avoid URL redirect
```suggestion
url: 'https://vincenttam.gitlab.io/gl-jekyll'
```

---

Test discussion.  Maybe it's better to Go Hugo.
```suggestion
test-another-line: 'whatever'
url: 'https://vincenttam.gitlab.io/gl-hugo'
```